src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/deconvolution/deconvolution_kernel_b_fs_zyx_fsv16.cpp

```diff
@@ -283,4 +275,51 @@ JitConstants DeconvolutionKernel_b_fs_zyx_fsv16::GetJitConstants(const deconvolu
     return jit;
 }
 
+KernelsData DeconvolutionKernel_b_fs_zyx_fsv16::GetKernelsData(const Params& params, const optional_params& options) const {
+    assert(params.GetType() == KernelType::DECONVOLUTION);
+
+    if (!Validate(params, options)) {
+        return{};
+    }
+
+    const deconvolution_params& orgParams = static_cast<const deconvolution_params&>(params);
```

Comment:
If most of the codes are from the parent class except the can_use_memory set, how about refactor them as Parent::GetKernelsData(...) ?

Reply:
Done. Thanks!